pkg/ext-proc/backend/inferencemodel_reconciler_test.go

```diff
@@ -146,3 +146,24 @@ func populateServiceMap(services ...*v1alpha1.InferenceModel) *sync.Map {
 	}
 	return returnVal
 }
+
+func mapsEqual(map1, map2 *sync.Map) bool {
```

Comment:
is this needed somewhere?

Reply:
yes, this was moved from the endpointslice test file, and it was used in anther test file in the same pkg